server/models/TripModel.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require('mongoose')
+
+const tripSchema = new mongoose.Schema({
+  _id: {
+    type: String,
+    required: true,
+  },
+  tripName: String,
+  destination: String,
```

Comment:
should more of these fields be required?

Reply:
ill look into it, which ones specifically?

Reply:
Good question...maybe match the form schema? If there is one? Otherwise prolly ok for now, it would be quick change in the future